stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -673,73 +583,68 @@ end
 #! via the advice stack.
 #! The signature is valid if and only if the procedure returns.
 #!
-#! Input: [PK, MSG, ...]
-#! Output: [...]
+#! Inputs:
+#!   Operand stack: [PK, MSG, ...]
+#!   Advice stack:  [tau0, tau1, h_0, ..., h_511, s2_0, ..., s2_511, pi_0, ..., pi_1022, nonce0, ..., nonce7]
+#! Outputs:
+#!   Operand stack: []
+#!   Advice stack:  []
+#!
+#! where:
+#! - h_i are the coefficients of the expanded public key polynomial.
+#! - s2_i are the coefficients of the signature polynomial.
+#! - pi_i are the coefficients of `h * s2` in Z_Q[x] where Q is the Miden VM prime.
+#! - nonce_i are field elements representing the nonce associated to the signature.
```

Comment:
Should we also describe what "tau" is here?

Reply:
Added a short description